modules/logging-release-notes-6-0-0.adoc

```diff
@@ -0,0 +1,121 @@
+// module included in logging-6-0-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-6-0-0_{context}"]
+= Logging 6.0.0
+
+This release includes link:https://access.redhat.com/errata/RHBA-2024:6693[OpenShift Logging Bug Fix Release 6.0.0]
+
+[id="logging-release-notes-6-0-0-compatibility-support-matrix"]
+= Compatibility and support matrix
+
+In the table, components are marked with the following statuses:
+
+[horizontal]
+TP:: Technology Preview
+GA:: General Availability
+
+The components or features in link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] status are experimental and are not intended for production use.
+
+[NOTE]
+====
+To know about compatible {product-title} versions for {logging-uc} releases, see link:https://access.redhat.com/product-life-cycles?product=Red%20Hat%20OpenShift%20Logging[Product Life Cycles].
+====
+
+.Components support matrix
+[options="header"]
+|===
+
+| {logging-uc} Version 6+| Component Version
+
+| Operator | `eventrouter` | `logfilemetricexplorer` | `loki` | `lokistack-gateway` | `opa-openshift` | `vector`
+
+|6.0 | 0.4 (GA) | v1.1 (GA) | v3.1.0 (GA) | v0.1 (GA) | v0.1 (GA) | v0.37.x (GA)
+
+|===
+
+[id="logging-release-notes-6-0-0-removal-notice"]
+== Removal notice
+
+In Logging 6.0, the support is removed for:
+
+* Red Hat managed Elasticsearch log storage solution. For example, elasticsearch-operator
+* Red Hat managed log visualization using Kibana and Console Log View Plugin
+* Fluentd-based log collector implementations
++
+[NOTE]
+====
+The log collector deployments based on Fluentd will migrate to Vector
+====
+* ClusterLogging.logging.openshift.io custom resource
+* ClusterLogForwarder.logging.openshift.io custom resource
```

Comment:
These CR bullets get lost under the title in my opinion and lose their meaning.    can we add (replaced by single "observability" resource)?  or similar?   or, at least a clarifying statement that details the result of this "removal"

Reply:
@cahartma Can you please elaborate upon the change? The bullet covers the following jiras: [https://issues.redhat.com/browse/LOG-5368](https://www.google.com/url?q=https://issues.redhat.com/browse/LOG-5368&sa=D&source=docs&ust=1726987835120699&usg=AOvVaw1si58vTpNdxdMIDLDDHw81) and [https://issues.redhat.com/browse/LOG-5803](https://www.google.com/url?q=https://issues.redhat.com/browse/LOG-5803&sa=D&source=docs&ust=1726987835120787&usg=AOvVaw3nKqMOmGH_0-rRKBgAV6rm)
I have used the content from the release notes text. Do let me know what else do you need added in here.

Reply:
@cahartma If the bullet points were causing confusion, i have removed it and put in sentences instead.